### ISSUE TITLE
Rule proposal: `no-array-unshift-unshift`

### ISSUE THREAD

**Issue by B2o5T** [sindresorhus/eslint-plugin-unicorn#1865]
### Description

`unshift` is like `push` accept multiple arguments.

### Fail

```js
let arr = [];
arr.unshift('foo', 2)
arr.unshift(true)
```

### Pass

```js
let arr = [];
arr.unshift(true, 'foo', 2)
```

### Additional Info

The error message should contain a notice that all elements from the second `unshift` call should be placed first.

**Comment by fisker** [comment]
https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1019#issue-785860890, seems no one interested.

**Comment by fregante** [comment]
Duplicate of https://github.com/sindresorhus/eslint-plugin-unicorn/issues/1019
